pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -404,6 +408,18 @@
   front_join_us_challenge: "Win a video chat for your classroom with a celebrity!"
   front_join_us_learn_more: "Learn more"
 
+  cta_see_whats_new: "See what's new"
+  cta_try_more_activities: 'Try more activities'
+  cta_try_an_activity: 'Try an activity'
+
+  hoc_homepage_join_the_movement_heading: 'Join the movement'
+  hoc_homepage_join_the_movement_description: 'Hour of Code is available year round, but every year in December your class can join millions of students around the world celebrating Computer Science Education Week with the Hour of Code. Registration for the annual celebration starts each year in October.'
+  hoc_homepage_what_is_hoc_subhead: 'One-hour tutorials in over 45 languages. No experience needed. Hour of Code activities are available for free year-round.'
+  hoc_homepage_what_is_hoc_description: 'The Hour of Code is a one-hour introduction to computer science, using fun tutorials to show that anybody can learn the basics. This grassroots campaign is supported by over 400 partners and 200,000 educators worldwide.'
+  hoc_homepage_what_is_hoc_cta_how_to_guide: "or <a href='%{howto_guide_url}'>read our How-to guide</a> to plan an event for your class"
+  hoc_homepage_highlights_heading: 'Hour of Code highlights'
+  hoc_homepage_organized_by: 'The Hour of Code is organized by'
```

Comment:
This will be followed by the Code.org logo:
<img width="371" alt="Screen Shot 2022-09-20 at 12 14 03 PM" src="https://user-images.githubusercontent.com/9256643/191344651-0a84ff54-46b1-477e-9468-8146f8c20e4c.png">

Reply:
This isn't a definite yet either (we're waiting on Cameron's approval) …I'll make this a full sentence: `The Hour of Code is organized by [Code.org](%{codeorg_url}).` and include the logo underneath if we add this to the page.

Reply:
Has Cameron approved?

Reply:
Not yet, but it'll be quick to add if/when he does.